test/heartbeat_test.go

```diff
@@ -0,0 +1,93 @@
+package test_test
```

Comment:
why is this package `test_test`?

Reply:
indeed this is an integration-test. Which package do you think is better?

Reply:
Oh I see, the test is put in a separated package.